popup.js

```diff
@@ -0,0 +1,41 @@
+const close1 = document.querySelector('.close1');
+const close2 = document.querySelector('.close2');
+const close3 = document.querySelector('.close3');
+const close4 = document.querySelector('.close4');
+
+// card 1
+document.querySelector('.see-pro1').addEventListener('click', () => {
+  document.querySelector('.pop-1').classList.toggle('active');
+  // document.querySelector('.overlay').classList.toggle('active');
+});
+
+close1.addEventListener('click', () => {
+  document.querySelector('.pop-1').classList.toggle('active');
+});
+
+// card 2
+document.querySelector('.see-pro2').addEventListener('click', () => {
+  document.querySelector('.pop-2').classList.toggle('active');
+});
+
+close2.addEventListener('click', () => {
+  document.querySelector('.pop-2').classList.toggle('active');
+});
+
+// card 3
+document.querySelector('.see-pro3').addEventListener('click', () => {
+  document.querySelector('.pop-3').classList.toggle('active');
+});
+
+close3.addEventListener('click', () => {
+  document.querySelector('.pop-3').classList.toggle('active');
+});
+
+// card 4
+document.querySelector('.see-pro4').addEventListener('click', () => {
+  document.querySelector('.pop-4').classList.toggle('active');
```

Comment:
- [ ] Good job for creating the popup but I see here that you display the data from your HTML code while it would be better if you use an object to display the items in each popup like this 

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
the reason for this is when you try to add a real project to your portfolio it would be easier to just change the data from your object not to change the data in the Html code.